ingest/src/test/java/com/microsoft/azure/kusto/ingest/E2ETest.java

```diff
@@ -39,7 +39,7 @@
     private static final String databaseName = System.getenv("TEST_DATABASE");
     private static final String appId = System.getenv("APP_ID");
     private static final String appKey = System.getenv("APP_KEY");
-    private static final String tenantId = System.getenv("TENANT_ID");
+    private static final String tenantId = System.getenv().getOrDefault("TENANT_ID","microsoft.com");
```

Comment:
This is good, but FYI the auth methods can handle a null authorityId:
    if (authorityId == null) {
        authorityId = ORGANIZATION_URI_SUFFIX;
    }